AaxDecrypter/NFO.cs

```diff
@@ -3,57 +3,50 @@ namespace AaxDecrypter
 {
     public static class NFO
     {
-        public static string CreateContents(string ripper, TagLib.File aaxcTagLib, ChapterInfo chapters)
+        public static string CreateContents(string ripper, AaxcTagLibFile aaxcTagLib, ChapterInfo chapters)
         {
-            var tag = aaxcTagLib.GetTag(TagLib.TagTypes.Apple);
-
-            string narator = string.IsNullOrWhiteSpace(aaxcTagLib.Tag.FirstComposer) ? tag.Narrator : aaxcTagLib.Tag.FirstComposer;
-
             var _hours = (int)aaxcTagLib.Properties.Duration.TotalHours;
             var myDuration
-                = (_hours > 0 ? _hours + " hours, " : "")
+                = (_hours > 0 ? _hours + " hours, " : string.Empty)
```

Comment:
"" <==> string.Empty . Let's not get in this battle. There are no winners.

Reply:
Fair enough! For my eyes, the most beautiful code has no string literals.

Reply:
ahem

> private static ReadOnlyByteVector narratorType = new ReadOnlyByteVector(0xa9, (byte)'n', (byte)'r', (byte)'t');
> private static ReadOnlyByteVector descriptionType = new ReadOnlyByteVector(0xa9, (byte)'d', (byte)'e', (byte)'s');
> private static ReadOnlyByteVector publisherType = new ReadOnlyByteVector(0xa9, (byte)'p', (byte)'u', (byte)'b');

:)  Just ribbing you. There's clearly no end to ugly literals throughout my code. Even when I put on my purity hat though I consider "" vs string.Empty a style preference, not a magic string since it's a very specific exception and isn't variable by localization.

Reply:
Don't look at it!